modules/apps/jaggery/emm/src/emm/units/user-listing/public/js/user-listing.js

```diff
@@ -141,6 +141,75 @@ function removeUser(uname, uid) {
 }
 
 /**
+ * Following click function would execute
+ * when a user clicks on "Reset Password" link
+ * on User Listing page in WSO2 MDM Console.
+ */
+function resetPassword(uname) {
+    var username = uname;
```

Comment:
No need to assign the parameter value again to another local variable. uname is already a local variable.